test/e2e/common/container_probe.go

```diff
@@ -50,9 +50,9 @@ var _ = framework.KubeDescribe("Probing container", func() {
 	})
 
 	/*
-		    Testname: pods-readiness-probe-initial-delay
-		    Description: Make sure that pod with readiness probe should not be
-			ready before initial delay and never restart.
+		Release : v1.9
+		Testname: Pod readiness probe with initial delay
```

Comment:
based on formatting of your other test cases I would expect this to be "Pod readiness probe, with initial delay"

Reply:
Yes, Fixed.